evaluation/benchmarks/for-loops/bio/100G.txt

```diff
@@ -0,0 +1,26 @@
+CHS HG00614 ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data/HG00614/cg_data/HG00614_lcl_SRR821831.mapped.COMPLETE_GENOMICS.CGworkflow2_2_evidenceOnly.CHS.high_coverage.20130401.bam 
```

Comment:
Is this an input log or used to download inputs?

Reply:
This is used to download the inputs

Reply:
used to download inputs